packages/azure-tracing/src/azure-functions.mts

```diff
@@ -0,0 +1,26 @@
+import { createAddHookMessageChannel } from "import-in-the-middle";
+import { register } from "module";
+
+const { registerOptions, waitForAllMessagesAcknowledged } =
+  createAddHookMessageChannel();
+
+register("import-in-the-middle/hook.mjs", import.meta.url, registerOptions);
+
+import { AzureFunctionsInstrumentation } from "@azure/functions-opentelemetry-instrumentation";
+import { useAzureMonitor } from "@azure/monitor-opentelemetry";
+import { metrics, trace } from "@opentelemetry/api";
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { UndiciInstrumentation } from "@opentelemetry/instrumentation-undici";
+
+useAzureMonitor();
+
+registerInstrumentations({
+  instrumentations: [
+    new UndiciInstrumentation(),
+    new AzureFunctionsInstrumentation({ enabled: true }),
```

Comment:
suggestion: I won't include this until they fix [this behavior](https://github.com/Azure/azure-functions-nodejs-opentelemetry/issues/8)

You can obtain the same without verbose debug logging using this commented 
snippet here https://pagopa.github.io/dx/docs/articles/azure-tracing#example-integration-with-app-service